models/user.js

```diff
@@ -10,12 +10,12 @@ const userSchema = new Schema({
   password2: { type: String },
   createdDate: { type: Date, default: Date.now },
   token: { type: String },
-  isOwner: { type: Boolean, default: false },
   role: {
     type: String,
-    enum: ['user', 'owner', 'admin'],
+    enum: ['user', 'owner', 'moderator', 'admin'],
     default: 'user',
   },
+  reviews: [{ type: Schema.Types.ObjectId, ref: 'Review' }],
```

Comment:
Hi @cynthiaeddy! You added the reviews collection reference to the business modal using `reviews: [{ type: ObjectId, ref: 'reviews' }],` I am curious if there was a reason you referenced the same collection differently at this point? Thank you in advance for your time!

Reply:
not quite there yet as far as getting the reviews to show up in the business collection, I'm thinking it should be reviews: [{ type: ObjectId, ref: 'reviews' }] but was playing with it and shouldn't affect this pr.